src/back2top.less

```diff
@@ -0,0 +1,20 @@
+.to-top {
+    opacity: 1;
+    display: inline-block;
+    padding: 1em;
+    position: fixed;
+    bottom: 1em;
+    right: 1em;
+    .paper-btn {
+    padding: 0.6em 1em;
+    background: white;
```

Comment:
Could this use the variable instead of hardcoded value

Reply:
variable for color after defining in colors.less, right?

Reply:
White color is not defined in colors.less. Also this is used in container.less 
```
.paper {
background: #FFF
}
```

Reply:
Done